tests/queries/0_stateless/02932_kill_query_sleep.sh

```diff
@@ -10,7 +10,7 @@ function wait_query_started()
     local query_id="$1"
     timeout=60
     start=$EPOCHSECONDS
-    while [[ $($CLICKHOUSE_CLIENT --query="SELECT count() FROM system.processes WHERE query_id='$query_id'") == 0 ]]; do
```

Comment:
With query cache enabled this query `SELECT count() FROM system.processes WHERE query_id='$query_id'` was cached and the previous result was used. Which is not what we want in this test. See [failure](https://s3.amazonaws.com/clickhouse-test-reports/58893/863c6523a301077bc87b8e75c4dcaf5882bb33b9/stress_test__asan_.html).

Reply:
I understand that this is a bug in query cache. `system.processes` should never be cached. Let's report it and fix it